### ISSUE TITLE
`latest` build is using very old commit

### ISSUE THREAD

**Issue by cheerfulstoic** [rtfd/readthedocs.org#1210]
This is on the `neo4jrb` project:

https://readthedocs.org/projects/neo4jrb/

Here's a relevant build:

https://readthedocs.org/builds/neo4jrb/2505867/

It's using the changeset `e6a5ba2e2d87fa0bcb974cf5bfd1cb040532fee7`.  This changeset is from May 2011 and it was previously pointed to by the `latest` tag in our git repository.  I've since deleted the tag to try to fix this problem.  If I delete and re-create the project it seems to work again, but only for a couple of builds and then it reverts to trying to use this old commit.  It's supposed to use the `HEAD` of `master`, right?

**Comment by dlfryar** [comment]
I'm seeing a similar behavior in other projects.  However, deleting the project and deleting the build dirs before the project didn't help either.

http://docs.readthedocs.org/en/latest/faq.html#deleting-a-stale-or-broken-build-environment

It appears that there is a stale cache somewhere and previous versions get picked up.  If you create a new repo with the same files and create a new project everything works correctly.

**Comment by cheerfulstoic** [comment]
Yeah, I needed to go to that URL to clear clear the build directory.  I actually needed to do it a couple of times and it still sometimes seems to get stuck on an old build (I haven't check for a little while, though)

**Comment by gregmuellegger** [comment]
@cheerfulstoic Is this still an issue?

**Comment by cheerfulstoic** [comment]
I'm going to say no, somewhat hesitantly ;)

There have been times when I've needed to run the wipe at `https://readthedocs.org/wipe/neo4jrb/latest/` multiple times (which now has a UI where it didn't before, which is great).  I also discovered that I had to delete the tag again in git recently because it was apparently still there (maybe I didn't do it correctly, but I think I saw it missing it GitHub.  Anyway...).

It has gone successfully for a few days and failed in the past, but it seems to have been good for the last week or two, so I'll close this issue unless something comes up ;)

Thanks!
